packages/@expo/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -307,6 +307,11 @@ export function getDefaultConfig(
         .filter((assetExt: string) => !sourceExts.includes(assetExt)),
       sourceExts,
       nodeModulesPaths,
+      blockList: [
+        // .expo/types contains generated declaration files which are not and should not be processed by Metro.
+        // This prevents unwanted fast refresh on the declaration files changes.
+        /\.expo[\\\/]types/, //eslint-disable-line no-useless-escape
```

Comment:
fwiw The ignore probably triggers here for `[\/]` since that's not needed in a character group

Reply:
Thanks, yes, that was it.